backend/spec/requests/api/v1/projects_spec.rb

```diff
@@ -90,14 +91,25 @@
           end
         end
 
-        context "when filtered by searched text" do
+        context "when filtered by project name" do
           let("filter[full_text]") { @project.name }
 
           it "contains only correct records" do
             expect(response_json["data"].pluck("id")).to eq([@project.id])
           end
         end
 
+        context "when filtered by partial project name" do
+          before(:each) do
```

Comment:
this is detail, but the most of the code at this repo puts `let` before `before`. Could we stick to this? :)

Reply:
yes, I try to stick to the convention, but in this case how would you use `let` for stubbing? when there's no result to be memoized, to me `before` feels more natural

Reply:
I am fine with using before @agnessa :), but tests at this project contain first `let` and then `before` blocks so in this case it should look like:
```
let(:filters) { {full_text: "ba", language: :en} }
let!(:correct_project_en) { create :project, name_en: "yellow banana", language: :en }
let!(:correct_project_es) {
  create :project, name_es: "plátano amarillo", name_en: "yellow banana", language: :es
}
let!(:incorrect_project) { create :project, name_en: "red apple", language: :en }

before do
  allow(Project).to receive(:translatable_attributes).and_return([:name])
end
```
but you know, it is detail. For me it is easier to read when scanning code when `let`, `before`, etc. are always at same places and there is ideally extra space between these two sections.

Reply:
ah, that's what you meant! 🤦‍♀️ ☕